sdks/python/test-suites/portable/py2/build.gradle

```diff
@@ -113,28 +98,6 @@ task crossLanguagePortableWordCount {
   }
 }
 
-task postCommitPy2IT {
-  dependsOn 'installGcpTest'
-  dependsOn 'setupVirtualenv'
-  dependsOn ':runners:flink:1.10:job-server:shadowJar'
-
-  doLast {
-    def tests = [
-        "apache_beam.io.gcp.bigquery_read_it_test",
-    ]
-    def testOpts = ["--tests=${tests.join(',')}"]
-    def cmdArgs = mapToArgString([
-        "test_opts": testOpts,
-        "suite": "postCommitIT-flink-py2",
-        "pipeline_opts": "--runner=FlinkRunner --project=apache-beam-testing --environment_type=LOOPBACK --temp_location=gs://temp-storage-for-end-to-end-tests/temp-it",
-    ])
-    exec {
-      executable 'sh'
-      args '-c', ". ${envdir}/bin/activate && ${runScriptsDir}/run_integration_test.sh $cmdArgs"
-    }
-  }
-}
-
 task crossLanguageTests {
```

Comment:
CrossLanguage tests, SparkVR tests, also need to be configured to run against one of high-priority versions, looks like currently they are hardcoded to run in Python 2. This can be also done in a future change.  Feel free to open a JIRA to to track this cleanup.

Reply:
Exactly. 
BEAM-9980 aims to do that.